public/resources/casestudies/adeo/asyncapi.yaml

```diff
@@ -0,0 +1,303 @@
+asyncapi: 2.4.0
+info:
+  title: Adeo Async API Case Study
+  version: "%REPLACED_BY_MAVEN%"
+  description: >
+    This Adeo specification illustrates how ADEO uses Async API to document some of their exchanges
+  contact:
+    name: AsyncApi team
```

Comment:
```suggestion
    name: AsyncAPI team
```

Reply:
actually better `AsyncAPI Community`. I'll fix that

Reply:
good catch!!